examples/benchmark_stem_image.py

```diff
@@ -0,0 +1,36 @@
+import click
+import glob
+from stempy import io, image
+import sys
+import time
+
+# Beware of disk caching when using this benchmark
+@click.command()
+@click.argument('path', type=click.Path(exists=True, file_okay=False))
+@click.option('-n', '--num-runs', default=10, help='Number of runs to perform')
+def run_benchmarks(path, num_runs):
+    """Run benchmarks on a directory containing data*.dat files"""
+    files = []
+    for f in glob.glob(path + '/data*.dat'):
```

Comment:
I was thinking that we should be able to pass in the whole glob expression, not sure we can rely on the file name starting with data.